e2e_testing/torchscript/elementwise.py

```diff
@@ -325,23 +325,40 @@ def ElementwiseMaximumModule_basic(module, tu: TestUtils):
 
 # ==============================================================================
 
-
-class ElementwiseGtScalarModule(torch.nn.Module):
+class ElementwiseGtScalarFloatModule(torch.nn.Module):
```

Comment:
nit: it seems like the convention below is to do `FloatScalar` rather than `ScalarFloat`